diff --git a/kernel/kpm/super_access.c b/kernel/kpm/super_access.c
index b3e72237..cdc484a0 100644
--- a/kernel/kpm/super_access.c
+++ b/kernel/kpm/super_access.c
@@ -171,9 +171,9 @@ DYNAMIC_STRUCT_BEGIN(task_struct)
     DEFINE_MEMBER(task_struct, group_leader)
     DEFINE_MEMBER(task_struct, mm)
     DEFINE_MEMBER(task_struct, active_mm)
-#if LINUX_VERSION_CODE > KERNEL_VERSION_4_10
-    DEFINE_MEMBER(task_struct, thread_pid)
-#endif
+//#if LINUX_VERSION_CODE > KERNEL_VERSION_4_10
+//    DEFINE_MEMBER(task_struct, thread_pid)
+//#endif
     DEFINE_MEMBER(task_struct, files)
     DEFINE_MEMBER(task_struct, seccomp)
 #ifdef CONFIG_THREAD_INFO_IN_TASK
